python/ray/data/datasource/__init__.py

```diff
@@ -1,3 +1,5 @@
+from iceberg_datasource import IcebergDatasource
```

Comment:
Will fix this - for now I was unable to do a pip editable install on my machine, so just had this for testing